### ISSUE TITLE
Pip installed CLI tool not in PATH

### ISSUE THREAD

**Issue by matthias-Q** [nix-community/dream2nix#904]
Hi,
I want to use nix flakes for a develop environment. The cli tool (dagster) can be installed via pip. I am able to set everything up - and the python library part can be imported correctly. However, the dagster cli command cannot be found. Tab completion however suggest dagster. 
I haave pushed by flakes + lockfiles in a repo: https://github.com/matthias-Q/nixgster

Thanks,
Matt

**Comment by rupurt** [comment]
@matthias-Q I worked around a similar issue for other tools. I just manually added the app to the scripts section in pyproject.toml

**Comment by matthias-Q** [comment]
Not sure I understand. Can you give me an example? I always thought that in the scripts section, there has to be a `<module>.<file>:function` defined as a script. Dagster installs multiple cli tools.

**Comment by rupurt** [comment]
Yeah, that's what I've defined. Haven't done it for Dagster though...
